tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/GitStore.cs

```diff
@@ -246,7 +246,7 @@ private void Clean(GitAssetsConfiguration config)
         {
             try
             {
-                GitHandler.Run("checkout *", config);
+                GitHandler.Run("checkout .", config);
```

Comment:
`config` brings the working directory to `GitHandler.Run`. So we're invoking `git checkout .` at repo root. Safe crossplat.